### PR TITLE
Sunday pull request #1

### PR DESCRIPTION
**Pull request by nathan-dunn**
Added a bunch

### CODE REVIEW

File: server/routes/matches/matches.js [hunk 1]
```diff
@@ -3,22 +3,37 @@ const db = require('../../models');
...
         }
       })
+      .then(matches => {
+        dogObjs = matches;
+        let promises = dogObjs.map(dogObj => {
+          let score = dogObj.dataValues.score;
+
+          return db.Dog.findOne({
+            where: {
+              id: dogObj.dataValues.dog
```
Comment (max-hoffman) [inline server/routes/matches/matches.js]: it's one phrase, this can be a one-liner

File: server/routes/matches/matches.js [hunk 2]
```diff
@@ -3,22 +3,37 @@ const db = require('../../models');
...
+            }
+          });
+        });
+        return Promise.all(promises);
+      })
       .then(results => {
-        res.status(200).json(results)
```
Comment (max-hoffman) [inline server/routes/matches/matches.js]: i think this can be cleaned up a little

**Comment by nathan-dunn** [comment]
Yes, but I was trying to smuggle the scores into the promises somehow so
rearranged some things.

**Comment by nathan-dunn** [comment]
I think I said it wrong.  I'm still writing it.  It grabs all the dog names
just fine.  But it needs to do more than that, so I'm working on that as we
speak.  I would love for it to be all nice and tidy and ES6 as soon as it
does everything I need.

**Comment by max-hoffman** [comment]
do you want me to leave this open for a little? also we can move all of the dog images into dropbox if you want

**Pull request merged**